<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="SongsList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSong">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSong">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="SongNameLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>歌名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="SingerNameLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>歌手名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SongName">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="SingerName">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="GetRandom">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>随机</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CopyBoard">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="CopyBoardLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剪贴板：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MakeRandom">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打乱</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="MainMenu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="ActionFileIn"/>
    <addaction name="ActionFileOut"/>
    <addaction name="ChangeFile"/>
   </widget>
   <addaction name="MainMenu"/>
  </widget>
  <action name="ActionFileIn">
   <property name="text">
    <string>导入点歌单文件</string>
   </property>
  </action>
  <action name="ActionFileOut">
   <property name="text">
    <string>导出点歌单文件</string>
   </property>
  </action>
  <action name="ChangeFile">
   <property name="text">
    <string>旧版转换新版</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>SongsList</tabstop>
  <tabstop>SongName</tabstop>
  <tabstop>SingerName</tabstop>
  <tabstop>AddSong</tabstop>
  <tabstop>DeleteSong</tabstop>
  <tabstop>CopyBoard</tabstop>
  <tabstop>MakeRandom</tabstop>
  <tabstop>GetRandom</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
